src/nfa/rdfa.cpp

```diff
@@ -44,10 +44,9 @@ void raw_dfa::stripExtraEodReports(void) {
 }
 
 bool raw_dfa::hasEodReports(void) const {
-    for (const dstate &ds : states) {
-        if (!ds.reports_eod.empty()) {
-            return true;
-        }
+    auto rempt = [](const dstate &d) { return !d.reports_eod.empty(); };
```

Comment:
you could use the same name here 'isempty' as above.

Reply:
done